### DISCUSSION TITLE
How does it work with GroundingDINO I noticed in the latest update

### DISCUSSION THREAD

**Discussion by lefthgir** [continue-revolution/sd-webui-segment-anything#23] [Q&A]
Now I am experimenting with this extention in stable diffusion, it works damn good.Howerver, just being curious, how to apply the detecting function to accurate the chosen object under the current interface of Segment-anything.

**Comment by continue-revolution** [comment]
Either you `cd` to the directory where this extension lives in and run `git checkout GroundingDINO` on your terminal, or you wait probably some hours for me to merge #21 to the master branch. I'm now starting to work on merging. Stay tuned!

**Reply by lefthgir** [reply]
Thx for replying. Surely, I prefer working with the extention merged to building up a new required environment for GroundingDINO token labeling function. Blessing!

**Comment by continue-revolution** [comment]
Master branch merged via merging #21 

Check it out by shooting a `git pull` on your terminal!
